CODE_OF_CONDUCT.md

```diff
@@ -30,6 +30,8 @@ advances
   address, without explicit permission
 * Other conduct which could reasonably be considered inappropriate in a
   professional setting
+* Attempting to contact maintainers outside of GitHub issues without a pre-existing
```

Comment:
Perhaps "outside of GitHub issues" &rarr; "outside of github.com" just to make it clear that issues, PRs, any other communication on github.com is ok?

Also rather than:

> without a pre-existing expectation of that contact (e.g., they asked you to email them)

perhaps:

> without an explicit invitation to do so

I'm also told that the Latin abbreviations such as i.e., e.g., _et al_, etc are opaque for people whose primary language is not English, so it is better to use their English equivalents when necessary.

Reply:
That's great feedback, thanks!